www/js/control/SettingRow.tsx

```diff
@@ -12,18 +12,24 @@ type Props = {
   descStyle?: any;
 };
 const SettingRow = ({ textKey, iconName, action, desc, switchValue, descStyle }: Props) => {
-  const { t } = useTranslation(); //this accesses the translations
-  const { colors } = useTheme(); // use this to get the theme colors instead of hardcoded #hex colors
+  const { t } = useTranslation(); // Access translations
+  const { colors } = useTheme(); // Get theme colors
 
   let rightComponent;
   if (iconName) {
-    rightComponent = <List.Icon icon={iconName} />;
+    rightComponent = <List.Icon icon={iconName} aria-hidden={true} />;
```

Comment:
I wonder if manually setting the accessibilityLabel to "" here would fix the double repetition issue. I would also suggest looking at the react native paper code (it is open-source!) to see if this has been reported by others and whether there are any workarounds.

Reply:
Lets figure this out in the subsequent PR. New issue created: https://github.com/e-mission/e-mission-docs/issues/1049